docs/dm/zh_CN/maintenance/metrics-alert.md

```diff
@@ -3,6 +3,14 @@ DM 监控与告警
 
 目前只有 dm-worker 提供了 metrics， dm-master 暂未提供。
 
+### 内容索引
+- [任务监控项](#Task)
+- [Relay Log Unit 监控项](#Relay-Log-Unit)
+- [Mydumper Unit 监控项](#Mydumper-Unit)
+- [Load Unit 监控项](#Load-Unit)
+- [Binlog-Replication-Unit 监控项](#Binlog-Replication-Unit)
```

Comment:
should we need to unify the name? (sync unit or binlog replication unit)

Reply:
it should be binlog replication unit, sync is not a good name